packages/mdc-select/README.md

```diff
@@ -194,19 +210,18 @@ seamless, un-invasive experience in browsers where a native select may be more a
 on a mobile device. It does not require any javascript, nor any CSS for `mdc-menu` or `mdc-list`.
 
 ```html
-<select class="mdc-select">
-  <option value="" default selected>Pick a food</option>
-  <option value="grains">Bread, Cereal, Rice, and Pasta</option>
-  <option value="vegetables">Vegetables</option>
-  <optgroup label="Fruits">
-    <option value="apple">Apple</option>
-    <option value="oranges">Orange</option>
-    <option value="banana">Banana</option>
-  </optgroup>
-  <option value="dairy">Milk, Yogurt, and Cheese</option>
-  <option value="meat">Meat, Poultry, Fish, Dry Beans, Eggs, and Nuts</option>
-  <option value="fats">Fats, Oils, and Sweets</option>
-</select>
+<div class="mdc-select">
```

Comment:
You might want to update the wording in the preceding paragraph, since I guess the `mdc-select` class does not work directly on the native `select` element, but rather wrapped around it.